tests/testthat/test_compare_raster_w_outputs.R

```diff
@@ -0,0 +1,154 @@
+context("Compare raster data to LDS outputs")
+library(dplyr)
+library(tibble)
+library(raster)
+library(rgdal)
+library(testthat)
+
+
+test_that("Compare raster outputs with LDS outputs at basin level", {
+  #Get land data from raster files for each basin intersection (32 regions, 235 basins)
+  #Get output data from LDS and compare with the above
+  #Allow an error tolerance of 2% to account for rounding. Note that this works fine with an error tolerance of 1%
+  
+  #1. Get LDS data and iso data
+  LDS_data<-read.csv("outputs/Land_type_area_ha.csv",skip = 9, header = T,stringsAsFactors = FALSE)
```

Comment:
For style: 

- all lower case variable names separated by an underscore
- spaces between variable and assignment as well as after commas and subsequent arguments